docs/users/api/playlist.rst

```diff
@@ -0,0 +1,9 @@
+Playlists
+=========
+
+The playlists API allows for the creation and editing of lists of recordings
+
+.. autoflask:: listenbrainz.webserver:create_app_rtfd()
+   :blueprints: playlist_api_v1
```

Comment:
we put the "/1/user/(playlist_user_name)/playlists" endpoint in the core blueprint, because we wanted it related to users, rather than under the /playlist namespace. However, from the perspective of a developer wanting to use the API, I think it makes sense to put all playlist endpoints in a single location.
Is there a way to individually add the other endpoints to here? (same with the user similarity endpoints)

Reply:
Yes, we can manually exclude and include endpoints. I'll do the needful changes.

Reply:
Where should we move the simlarity endpoints to?

Reply:
I was thinking of duplicating these endpoints - keep them in core (grouped by user-specific endpoints), but also additionally add them to playlist, so that if people go to the playlist section they'll also see them.

In that case, we don't need to move the similarity endpoints anywhere - just leave them in core if we don't have another place for them.